apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -13,13 +14,26 @@ import {
 import { AppRoute } from "~/libs/enums/enums.js";
 import {
 	useAppDispatch,
+	useAppForm,
 	useAppSelector,
 	useEffect,
 	useLocation,
 } from "~/libs/hooks/hooks.js";
 import { actions as userActions } from "~/modules/users/users.js";
 
+import { type Option } from "../select/select.js";
+
 const App = (): JSX.Element => {
+	type SelectData = {
+		options: Option<number>[];
+	};
+
+	const { control } = useAppForm<SelectData>({
+		defaultValues: {
+			options: [],
```

Comment:
let's pass some options to test
```suggestion
			options: ['Option 1', 'Option 2'],
```

Reply:
Vladyslav asked me to remove them in his previous review : ) but sure, I'll do it.